package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "joinmarket-webui",
+  "name": "Jam",
```

Comment:
```suggestion
  "name": "jam",
```
From [npm docs](https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields):
> The "name" field contains your package's name, and **must be lowercase** and one word, and may contain hyphens and underscores.

Reply:
Good point, updated. ✅